parastell/nwl_utils.py

```diff
@@ -112,42 +109,46 @@ def min_problem(theta, vmec, wall_s, phi, pt):
     # Compute first wall point
     fw_pt = np.array(vmec.vmec2xyz(wall_s, theta, phi))
     m2cm = 100
-    fw_pt = fw_pt*m2cm
-    
+    fw_pt = fw_pt * m2cm
+
     diff = np.linalg.norm(pt - fw_pt)
 
     return diff
 
 
-def find_coords(vmec, wall_s, phi, pt):
+def find_coords(data):
```

Comment:
this function takes a tuple so it plays nicely with `concurrent.futures.ProcessPoolExecutor`